internal/collect_es6_sources.bzl

```diff
@@ -40,11 +40,11 @@ def collect_es6_sources(ctx):
     if file.short_path.startswith(".."): continue
 
     rerooted_file = ctx.actions.declare_file(
-      "%s.es6/%s" % (
-        ctx.label.name,
+      "%s/%s" % (
         # the .closure.js filename is an artifact of the rules_typescript layout
         # TODO(mrmeku): pin to end of string, eg. don't match foo.closure.jso.js
-        file.short_path.replace(".closure.js", ".js")))
+        file.dirname,
```

Comment:
Tor had `file.dirname.replace("/external", "/node_modules"),` in his POC but the replace wasn't necessary. Not sure why it was needed before.

Reply:
I think there was a fix upstream that removed the need for this replace

Reply:
Thanks for the heads up. Was going to take a look at that today.